int32 count
int32 test_number
int32 dof
int32 channel

int32[] amplitudes
